src/constants/oauth2.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class CredentialType(Enum):
```

Comment:
Use standard values defined by Oauth2 flow

Reply:
It will be add later, when login route will create.